docs/evaluations.md

```diff
@@ -1,35 +1,99 @@
 # Evaluations
 
-## Schedule
+We use Discovery Engine's in-built evaluations feature to measure the quality and relevance of search results. This allows us to:
 
-When first created the crontasks had the following schedule:
+- [Monitor search quality](#How-evaluations-are-monitored) with automated alerts if relevance scores drop unexpectedly.
+
+- Compare search engines so that we can test out new configurations to improve search results.
+
+Evaluations rely on 'judgement lists' (a.k.a. ['sample query sets'][sample-query-sets] in Discovery Engine terminology). These are sets of search queries paired with targets and their ratings, indicating how relevant specific results are for those queries. We use three sets of judgement lists to create evaluations, which we refer to as ['datasets'](#Datasets). When evaluations are run, point-in-time search results are compared to the judgement lists to create [scores](#How-evaluations-are-scored), which can be tracked over time.
```

Comment:
Minor: The second sentence doesn't quite read right. I'm guessing the tense was changed at some point? 

Perhaps: "These are sets of search queries paired with targets and their ratings, that indicate how relevant specific results are for those queries"